docs/fundamentals/flow/gateway-args.md

```diff
@@ -6,6 +6,7 @@
 | `quiet` | If set, then no log will be emitted from this object. | `boolean` | `False` |
 | `quiet_error` | If set, then exception stack information will not be added to the log | `boolean` | `False` |
 | `timeout_ctrl` | The timeout in milliseconds of the control request, -1 for waiting forever | `number` | `60` |
+| `restart` | If set, the Executor will restart while serving if the YAML configuration source is changed. This differst from `reload` argument that this will restart the server and more configuration can be changed, like the number of replicas | `boolean` | `False` |
```

Comment:
```suggestion
| `restart` | If set, the Executor will restart while serving if the YAML configuration source is changed. This differs from `reload` argument in that this will restart the server and more configuration can be changed, like number of replicas | `boolean` | `False` |
```